src/vmshepherd/http/__init__.py

```diff
@@ -66,11 +66,11 @@ class Panel(web.View):
         runtime = shepherd.runtime_manager
         for name in presets:
             preset = shepherd.preset_manager.get_preset(name)
-            await preset.refresh_vms(if_older_than=60)
+            runtime_data = await runtime.get_preset_data(name)
             data['presets'][name] = {
                 'preset': preset,
-                'vms': preset.vms,
-                'runtime': await runtime.get_preset_data(name),
+                'vms': runtime_data.iaas['vms'] if 'vms' in runtime_data.iaas else [],
```

Comment:
I would recommend to use `Preset.get_vms` as the only one interface for that data, and yes - despite another  (redundant) call for runtime data. 

Current approach introduces two ways of doing it and exposes implementation details.

Reply:
Ok, right now vms in panel are taken from runtime, this avoids two reduntant callls.